sensor_msgs/Image descriptor
geometry_msgs/Point32 imageCoord
